arlo_server/__init__.py

```diff
@@ -341,19 +340,45 @@ def admin():
         return Response(result, content_type="text/plain")
 
 
+def require_audit_admin_for_organization(organization_id: Optional[str]):
```

Comment:
They're part of the python language's type annotation syntax. They aren't checked at runtime, but 3rd party tools can use them to do whatever they want. We use `mypy` to validate the types.

Reply:
Ooooh cool, so when do you run `mypy`?

We could potentially make aliases to `Any` for some of those other types so that we could at least notate it for ourselves to read.

Reply:
`mypy` runs with `make typecheck`. And I'm down for aliases to `Any` for types that are too "hard" to type properly.